readthedocs/settings/base.py

```diff
@@ -46,7 +46,7 @@ class CommunityBaseSettings(Settings):
     DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
 
     # Debug settings
-    DEBUG = True
+    DEBUG = False
```

Comment:
LOL, I keep doing this. But I also said to open a PR to make this a general setting.